features/fixtures/statistics.json

```diff
@@ -102,41 +105,52 @@
         "filterable": true,
         "preposition": "about"
       },
+      {
+        "key":"content_store_document_type",
+        "name":"Statistics",
+        "type":"radio",
+        "display_as_result_metadata":false,
+        "filterable":true,
+        "hide_facet_tag":true,
+        "preposition": "that are",
+        "option_lookup": {
+          "statistics_published": ["statistics", "national_statistics", "statistical_data_set", "official_statistics"],
+          "statistics_upcoming": ["statistics_announcement", "national_statistics_announcement", "official_statistics_announcement"],
+          "research": ["dfid_research_output", "independent_report", "research"]
+        },
+        "allowed_values": [
+          {
+            "label": "Statistics (published)",
+            "value": "statistics_published",
+            "default": true
+          },
+          {
+            "label": "Statistics (upcoming)",
+            "value": "statistics_upcoming"
+          },
+          {
+            "label": "Research",
+            "value": "research"
+          }
+        ]
+      },
       {
         "key": "organisations",
         "name": "Organisation",
         "short_name": "Organisation",
         "preposition": "from",
-        "type": "autocomplete",
+        "type": "text",
         "display_as_result_metadata": true,
         "filterable": true
```

Comment:
could you add `"show_option_select_filter": true` to this file?

This key has been added to govuk-content-schema so it should not break anything and the functionality will be added as part of https://github.com/alphagov/finder-frontend/pull/933

Reply:
@oscarwyatt You seemed to have missed off this second `"show_option_select_filter"` in your commit